core/templates/domain/exploration/ExplorationObjectFactory.ts

```diff
@@ -130,25 +133,29 @@ export class Exploration {
   }
 
   getInteractionCustomizationArgs(
-      stateName: string): InteractionCustomizationArgs {
+      stateName: string): InteractionCustomizationArgs | null {
     let interaction = this.getInteraction(stateName);
     if (interaction === null) {
       return null;
     }
     return interaction.customizationArgs;
   }
 
-  getInteractionInstructions(stateName: string): string {
+  getInteractionInstructions(stateName: string): string | null {
     let interactionId = this.getInteractionId(stateName);
-    return interactionId ? INTERACTION_SPECS[interactionId].instructions : '';
+    return interactionId ? INTERACTION_SPECS[
+      interactionId as InteractionSpecsKey
+    ].instructions : '';
   }
 
-  getNarrowInstructions(stateName: string): string {
+  getNarrowInstructions(stateName: string): string | null {
```

Comment:
Ditto as above.

Reply:
https://github.com/oppia/oppia/blob/8af8a439990f4ea692d9def3828fddc5b43747b0/extensions/interactions/interaction_specs.json#L48

The 'narrow_instructions' key is null for some objects in the interaction specs constant. Hence the return type is string | null.

Reply:
Again, this doesn't seem to be used anywhere.

Reply:
Can you also check the other methods in this file, since others also don't seem to be used.

Reply:
Done.